pkg/apis/garden/types.go

```diff
@@ -1308,6 +1308,15 @@ type Kubernetes struct {
 	Kubelet *KubeletConfig
 	// Version is the semantic Kubernetes version to use for the Shoot cluster.
 	Version string
+	// ClusterAutoscaler contains the configration flags for the Kubernetes cluster autoscaler.
+	ClusterAutoscaler *ClusterAutoscaler `json:"clusterAutoscaler,omitempty"`
+}
+
+// ClusterAutoscaler contains the configration flags for the Kubernetes cluster autoscaler.
+type ClusterAutoscaler struct {
+	// ScaleDownUtilizationThreshold defines the threshold in % under which a node is being removed
+	// +optional
+	ScaleDownUtilizationThreshold *float64
```

Comment:
What about validation for this field? Also, maybe we should use `IntOrString` for this as it's done e.g. for a deployment's `MaxUnavailable` field.

Reply:
Well actually this has to be a float.

Reply:
Well, you're completely right :see_no_evil: was too early for me in the morning. However, still, what about validation? Negative values don't make that much sense here.